flashcard_runner.rb

```diff
@@ -0,0 +1,19 @@
+require './lib/card'
+require './lib/turn'
+require './lib/deck'
+require './lib/round'
+
+@card_1 = Card.new("Broccoli, cabbage, and brussel sprouts are all the same species, True or False?", "True", :Biology)
```

Comment:
Is there a reason why all these variable in your runner file are all instance variables and not just normal variables?